08-Secure by default/Secure Templates/Bicep/EventHub.bicep

```diff
@@ -8,13 +8,16 @@ param eventHubName string
 param storageAccountNameForDiagnostics string
 param AuthorizationRules string = 'RootManageSharedAccessKey'
 
-resource eventHubNamespace_resource 'Microsoft.EventHub/namespaces@2017-04-01' = {
+resource eventHubNamespace_resource 'Microsoft.EventHub/namespaces@2022-01-01-preview' = {
   sku: {
     name: 'Standard'
     tier: 'Standard'
     capacity: 1
   }
   name: eventHubNamespace
+  properties: {
+       minimumTlsVersion: '1.2'
```

Comment:
Please add Conrol Name here too

Reply:
added the control id documentation here.